### ISSUE TITLE
How to actually run the tests

### ISSUE THREAD

**Issue by talamaska** [driftyco/ionic-app-lib#32]
I have forked and cloned the repository on Windows machine. I tried to run the tests, but I'm failing on serve.spec. So my question is, how we are supposed to run the test? Do we need to be inside some ionic project or those test should be able to run alone independently? Is there something else I can do to be able to run tests on Windows?

**Comment by leahciMic** [comment]
@talamaska The tests pass for me on mac and linux, are you able to use one of those? Or retry the tests, and for things like wrong slash, change the test to use `path.join`, or `path.normalize`.

**Comment by jbavari** [comment]
Yep, looks like I made a path mistake. Do you need help adjusting it? Fixing that path, may get it working again.

If you need help, holler at me.
